### PR TITLE
Naming convention: doc translation

### PR DESCRIPTION
**Pull request by innabelaya**

### CODE REVIEW

**Comment by innabelaya** [comment]
@tadatuta

File: method/naming/naming-convention.en.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+Let's say we have a CSS selector named `menuitemvisible`. A quick view of such a notation doesn't allow us to identify types of BEM entities from the name of the selector.
+
+Let’s add a delimiter:
+
+`menu-item-visible` or `menuItemVisible`
+
+Written like this, the name of the selector is clearly divided into logical parts. We can assume `menu` to be a block, `item` to be an element, and `visible` to be a modifier. However, real-life examples are often more complicated and not as straightforward, and that’s where the BEM naming convention comes in useful.
+
+The BEM methodology provides an idea for creating naming rules and implements that idea in its proprietary [CSS selector naming convention](#css-selector-naming-convention). However, a number of [alternative schemes](#alternative-naming-schemes) based on the BEM principles also exist in the world of web development.
```
Comment (tadatuta) [inline method/naming/naming-convention.en.md]: let's replace `proprietary` with something like `canonical`?

File: method/naming/naming-convention.en.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+
+CSS selector naming convention
+------------------------------
+
+-   Names of BEM entities are written using **numbers** and **lowercase** **Latin characters**.
```
Comment (tadatuta) [inline method/naming/naming-convention.en.md]: bold markup can be merged for `lowercase` and `Latin characters`

File: method/naming/naming-convention.en.md [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+```
+
+### Example of using the naming convention
+
+The implementation of an authorization form in HTML and CSS:
+
+*HTML*
+
+``` html
+<form class="form form_login form_theme_forest"> <input class="form__input"> <input class="form__submit form__submit_disabled"> </form>
```
Comment (tadatuta) [inline method/naming/naming-convention.en.md]: let's format code like this:

```html
<form class="form form_login form_theme_forest">
    <input class="form__input">
    <input class="form__submit form__submit_disabled">
</form>
```

**Comment by tadatuta** [comment]
wrote a few proposals

File: method/naming/naming-convention.en.md (hunk 1 above)
Comment (rasulksirov) [inline method/naming/naming-convention.en.md]: `canonical` sounds better indeed - will replace accordingly.

File: method/naming/naming-convention.en.md (hunk 2 above)
Comment (rasulksirov) [inline method/naming/naming-convention.en.md]: Yeah, it was just a copy-and-paste of the tags from the Russian version, where there's an `and` between the two phrases. Will merge them here.

**Comment by rasulksirov** [comment]
Thanks for the comments - I've updated the file in the ticket on ST, i.e. changes related to the first two comments above. As for the HTML formatting, I understand it must be a md file generation issue?

**Comment by innabelaya** [comment]
@rasulksirov Thank you! I will correct HTML formatting.

**Comment by innabelaya** [comment]
@tadatuta please check the corrections

**Comment by tadatuta** [comment]
:+1:

**Pull request merged**